tuner/examples/simple/simple_tuner.py

```diff
@@ -75,6 +75,7 @@ def main():
 
     path_config = libtuner.PathConfig()
     path_config.base_dir.mkdir(parents=True, exist_ok=True)
+    path_config._set_tune_output(Path("autotune_output.txt"))
```

Comment:
Also, can we give it a more descriptive name than `output`? It would not be clear to what to expect based on this name. Maybe `summary.log`?